ls1intum/Artemis
# [Chore] Rename Tutor Dashboard to Assessment Dashboard

### Motivation
The tutor dashboard name was misleading as it implies that only tutors should access it. Renaming it to assessment dashboard better reflects its function within Artemis. 

### Description
Rename Tutor Course Dashboard & Tutor Exam Dashboard => Assessment Dashboard. 
Tutor Exercise Dashboard => Exercise Assessment Dashboard

### Steps for testing
Click around the Artemis client. You should not see any translation errors, especially in the Assessment Dashboard
